### PR TITLE
content: source track v.next draft, address remainder of pre-merge issues

### PR DESCRIPTION
**Pull request by zachariahcox**
closes out the remainder of the pre-merge [issues](https://docs.google.com/document/d/13Xt8mA_2b00McGX2vkyhu4GQdFAqtXPu7YXE8ZA6ISE/edit?resourcekey=0-EqfHF79tUWAKp4PzsE3z1A#heading=h.au8zjzii8lgw). 

## changes

1. adds high-level document status section. 
2. add outstanding TODOs from ☝️ gdoc
3. add link to `label:source-track` issues in slsa repo
4. removes reference to "time" in the definition of "revision."
5. adds source track links to what's new file.

### CODE REVIEW

**Review by TomHennen — APPROVED** [review]
LGTM thanks!

**Comment by joshuagl** [comment]
> It looks like I technically have permission to merge this, but I don't think that's the right process.

Thanks for holding off! Per the CONTRIBUTING guide, content changes should have 3 approvers and allow a minimum of 72h for review: https://github.com/slsa-framework/slsa/blob/main/CONTRIBUTING.md#pull-request-types

File: docs/spec/v1.1/source-requirements.md [hunk 1]
```diff
@@ -1,22 +1,12 @@
 # SLSA Source Track
 
-## Objective
-
-The SLSA Source Track mitigates [Threat A ("Submit unauthorized change")](/spec/v1.0/threats#a-submit-unauthorized-change), scoped to a code repository and the organization that owns that repository. Concretely: an attacker must compromise the accounts of two organization members to publish code in a Source Level 3-conformant repository, and the evidence of those unauthorized changes cannot be destroyed without further attacks.
-
-## Changes from v0.1
+## Status: DRAFT
```
Comment (joshuagl) [inline docs/spec/v1.1/source-requirements.md:3]:
```suggestion
```
Let's drop this line, the whole document is already heavily indicated as a draft
Screenshot 2024-07-03 at 17 36 17

File: docs/spec/v1.1/source-requirements.md [hunk 2]
```diff
@@ -1,22 +1,12 @@
...
 
--   **Scope** The Source track is now scoped to Revisions rather than builds.
-Why?: To facilitate verification without anchoring it to a build.
+Open issues are tracked with the [source-track](https://github.com/slsa-framework/slsa/issues?q=is%3Aissue+is%3Aopen+label%3Asource-track) label in the [slsa-framework/slsa](https://github.com/slsa-framework/slsa) repository.
```
Comment (joshuagl) [inline docs/spec/v1.1/source-requirements.md:5]: Please leave todos within the doc, that better communicates to readers what's left to work on and makes it marginally easier to work through the todos when editing the file.

**Review by joshuagl — CHANGES_REQUESTED** [review]
Thanks, this LGTM but please restore the TODO items within the file contents.

File: docs/spec/v1.1/source-requirements.md (hunk 2 above)
Comment (zachariahcox) [reply docs/spec/v1.1/source-requirements.md:5]: the problem we were having was the todos were growing faster than our ability to approve changes to the draft. At best wouldn't it always be a snapshot of the issues in that query captured every time a pr is merged?

Comment (zachariahcox) [reply docs/spec/v1.1/source-requirements.md:5]: or @joshuagl to put it differently, which TODOs should appear in version control? there were previously only three, and there are currently 23 open issues. 

I would strongly suggest making this doc more accurate by tracking the current set of blocking issues. 

Maybe to your deep point, not all of those issues are blocking this doc from moving out of "draft" status, but I think that's something we should track and solve by labeling any such blocking issues as they appear?

What do you think?

Comment (TomHennen) [reply docs/spec/v1.1/source-requirements.md:5]: IMO the 'easy' thing is just to list the TODOs from https://docs.google.com/document/d/13Xt8mA_2b00McGX2vkyhu4GQdFAqtXPu7YXE8ZA6ISE/edit?resourcekey=0-EqfHF79tUWAKp4PzsE3z1A&tab=t.0 and then maybe also link to that tag as a helpful pointer.

File: docs/spec/v1.1/source-requirements.md [hunk 3]
```diff
@@ -1,22 +1,24 @@
...
+-   [] https://github.com/slsa-framework/slsa/issues/1070
+-   [] https://github.com/slsa-framework/slsa/issues/1071
+-   [] https://github.com/slsa-framework/slsa/issues/1072
+-   [] https://github.com/slsa-framework/slsa/issues/1074
+-   [] https://github.com/slsa-framework/slsa/issues/1075
+-   [] https://github.com/slsa-framework/slsa/issues/1076
+-   [] https://github.com/slsa-framework/slsa/issues/1077
+-   [] https://github.com/slsa-framework/slsa/issues/1078
+-   [] https://github.com/slsa-framework/slsa/issues/1079
+-   [] https://github.com/slsa-framework/slsa/issues/1080
```
Comment (joshuagl) [inline docs/spec/v1.1/source-requirements.md:17]:
```suggestion
-   [] [Structure & formatting don't match the build track](https://github.com/slsa-framework/slsa/issues/1069)
-   [] [Either identify the unique value of L1 or merge it with L2](https://github.com/slsa-framework/slsa/issues/1070)
-   [] [How to communicate SLSA source track metadata?](https://github.com/slsa-framework/slsa/issues/1071)
-   [] [Clarify source track objective](https://github.com/slsa-framework/slsa/issues/1072)
-   [] [Clarify the 'merger' identity in source track](https://github.com/slsa-framework/slsa/issues/1074)
-   [] [Flesh out the definition and bounds of 'identity', and why they're required](https://github.com/slsa-framework/slsa/issues/1075)
-   [] [VCS and SCP concerns are mixed or too prescriptive](https://github.com/slsa-framework/slsa/issues/1076)
-   [] [Clarify that self-hosted SCPs are allowed](https://github.com/slsa-framework/slsa/issues/1077)
-   [] [Create guidance for consumers on how to evaluate the source platform](https://github.com/slsa-framework/slsa/issues/1078)
-   [] [Clarify what must be retained during source migrations](https://github.com/slsa-framework/slsa/issues/1079)
-   [] [Refine requirements/guidance for trusted robots](https://github.com/slsa-framework/slsa/issues/1080)
```

**Review by joshuagl — APPROVED** [review]

**Review by lehors — APPROVED** [review]
LGTM

**Pull request merged**